components/mdns/tests/test_apps/pytest.ini

```diff
@@ -0,0 +1,24 @@
+[pytest]
```

Comment:
Please check if we can use one `ini` file for multiple components

Reply:
Indeed it worked.
I remember I tried it before and it wasn't supported yet, now it is possible